authors/apps/articles/serializers.py

```diff
@@ -0,0 +1,63 @@
+import re
+
+from rest_framework import serializers
+from .models import Article
+
+class ArticleSerializer(serializers.ModelSerializer):
+    """Serializers for creating and retrieving all articles"""
+    # Field in the database corresponding to the User model
+    author = serializers.SerializerMethodField()
+    # Uploads an image to the Cloudinary servers
+    images = serializers.ImageField(default=None)
+    
+
+    class Meta:
+        model = Article
+        fields = ['title',
+                  'slug',
+                  'description',
+                  'body',
+                  'created',
+                  'modified',
+                  'images',
+                  'author']
+        read_only_fields = ['created',
```

Comment:
Consider adding an `is_published` field to allow saving of drafts

Reply:
done